bazel/repository_locations.bzl

```diff
@@ -36,22 +53,26 @@ REPOSITORY_LOCATIONS = dict(
         #
         # chromium-81.0.4044.69
         urls = ["https://github.com/google/boringssl/archive/1c2769383f027befac5b75b6cedd25daf3bf4dcf.tar.gz"],
+        use_category = ["dataplane"],
     ),
     boringssl_fips = dict(
         sha256 = "b12ad676ee533824f698741bd127f6fbc82c46344398a6d78d25e62c6c418c73",
         # fips-20180730
         urls = ["https://commondatastorage.googleapis.com/chromium-boringssl-docs/fips/boringssl-66005f41fbc3529ffe8d007708756720529da20d.tar.xz"],
+        use_category = ["dataplane"],
     ),
     com_google_absl = dict(
         sha256 = "2693730730247afb0e7cb2d41664ac2af3ad75c79944efd266be40ba944179b9",
         strip_prefix = "abseil-cpp-06f0e767d13d4d68071c4fc51e25724e0fc8bc74",
         # 2020-03-03
         urls = ["https://github.com/abseil/abseil-cpp/archive/06f0e767d13d4d68071c4fc51e25724e0fc8bc74.tar.gz"],
+        use_category = ["dataplane"],
```

Comment:
Basically everywhere..

Reply:
Added comment that if dataplane or controlplane are set other uses are not needed to be declared, as it does not change security posture.